src/main/java/net/modfest/scatteredshards/api/ShardDisplaySettings.java

```diff
@@ -0,0 +1,80 @@
+package net.modfest.scatteredshards.api;
+
+import com.google.gson.JsonObject;
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.JsonOps;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+import net.minecraft.network.RegistryByteBuf;
+import net.minecraft.network.codec.PacketCodec;
+import net.minecraft.network.codec.PacketCodecs;
+import net.modfest.scatteredshards.api.impl.ColorCodec;
+
+public class ShardDisplaySettings {
```

Comment:
things that make more sense in a client config and you know this already and I maybe just need to get over our nepotism hesitation in stuffing kaleido into this mod but perfect for now